packages/smooth_app/lib/pages/product/edit_ingredients_page.dart

```diff
@@ -136,21 +145,14 @@ class _EditOcrPageState extends State<EditOcrPage> {
           _helper.getImageField().value == ImageField.PACKAGING.value
               ? ProductEditTask.packaging
               : ProductEditTask.ingredient,
+      widget: this,
     );
     final Product upToDateProduct = cachedProduct ?? changedProduct;
     await daoProduct.put(upToDateProduct);
     localDatabase.upToDate.set(upToDateProduct);
     if (!mounted) {
       return false;
     }
```

Comment:
We don't need this anymore if we don't show a SnackBar, do we

Reply:
@M123-dev You're right, there's room for simplification. Done in the next push.